CRM/Export/BAO/ExportProcessor.php

```diff
@@ -378,8 +378,14 @@ public function setAddresseeGreetingTemplate($addresseeGreetingTemplate) {
    * @param bool $isMergeSameHousehold
    * @param bool $isPostalableOnly
    * @param bool $isMergeSameAddress
-   */
-  public function __construct($exportMode, $requestedFields, $queryOperator, $isMergeSameHousehold = FALSE, $isPostalableOnly = FALSE, $isMergeSameAddress = FALSE) {
+   * @param array $formValues
+   *   Values from the export options form on contact export. We currently support these keys
+   *   - postal_greeting
+   *   - postal_other
+   *   - addresee_greeting
+   *   - addressee_other
+   */
+  public function __construct($exportMode, $requestedFields, $queryOperator, $isMergeSameHousehold = FALSE, $isPostalableOnly = FALSE, $isMergeSameAddress = FALSE, $formValues = []) {
```

Comment:
We haven't adopted an official documentation standard for arrays, but I like the looks of this one: https://stackoverflow.com/a/51032174/2918050